### ISSUE TITLE
MRA Alternatives are being downloaded always on full resync

### ISSUE THREAD

**Issue by aghus** [MiSTer-devel/Updater_script_MiSTer#73]
I have my update.ini with the option `MAME_ALT_ROMS=""`
according to what it says in the comments from the updater, it shouldn't download and extract the zip file with the alternatives MRA files.
Well, when doing a full resync it downloads the file and extract its contents on the `_Arcade/_alternatives` folder.

If after that i delete the `MRA-Alternatives_xxxxxxxx.zip` from `_Scripts\.mister_updater`, and also the `_alternatives` folder from `_Arcade` and perform and run the script again to do and optimized update then it does not download the file and behaves as expected.

**Comment by ash2hub** [comment]
> Am I correct to asume that you had this exact problem, and decided to add those two new lines to stop the updater from downloading the Alternative MRAs?

That is correct. I added those lines to keep the alternatives from being download at all.
